keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -207,54 +192,55 @@ Yes, the feature gate can be disabled, but Service resource that have set the ne
 
 * **What happens if we reenable the feature if it was previously rolled back?**
 
-New Services should be able to set the `trafficPolicy` field. Existing Services that have the field set already should not be impacted.
+New Services should be able to set the `internalTrafficPolicy` field. Existing Services that have the field set already should not be impacted.
```

Comment:
Actually, existing services with the field set will be impacted because their behavior will be reenabled (i.e. we will start enforcing local policy again).

Reply:
fixed, thanks